### DISCUSSION TITLE
Xdrip+ Not Working with Dexcom Share Invites

### DISCUSSION THREAD

**Discussion by andrewsawesome** [NightscoutFoundation/xDrip#2837] [Q&A] [answered]
I have two people following my Xdrip+ data via the Dexcom Share server. One is using Xdrip+ and another is using Sugarmate. 
The Sugarmate user hasn't been getting any data for the past few days.
When I went into the "Manage Followers" part of the app and tried to invite a new follower, after filling out the follower information, I always get "Failed to invite follower".
Share settings:
Dexcom USA based account: Checked
App version: e50dfdf-2023.05.03
Phone: Google Pixel 4a with May 5 2023 Security Update (Android 13)

**Comment by Navid200** [comment, marked as answer]
This feature is not currently getting lots of support from the developers.

Considering the master in your system is xDrip, is there a reason you don't use Nightscout?

The Nightscout support is very different because xDrip and Nightscout are both open source.  
Considering this alternative exists, there is not much incentive for developers to fix any issues that keep coming up when something on the Dexcom side is changed causing problems.

**Reply by andrewsawesome** [reply]
I used Dexcom Share because it was what I was using before. I also had iOS users using the Dexcom share app, but it seems the NightGuatd app is available on the iOS app store, so maybe I can tell them to use that.
Using Heroku for Nightscout is free, right?

**Reply by Navid200** [reply]
Not any more.  Heroku is not free as of November 2022.

But, there are free options.
This is one of several still free options:

https://navid200.github.io/xDrip/docs/Nightscout/GoogleCloud.html

**Reply by andrewsawesome** [reply]
Thanks, I used Google Cloud and the iOS user is following using the NightGuard app on iOS (Available on the app store). It seems to be working, and I like the fact that I can follow my BG on any browser. 
It was little hard to set up, but it seems like a good solution.

**Reply by Navid200** [reply]
If you already have Nightscout working, any follower using Android, can follow using xDrip and the following instructions:
https://navid200.github.io/xDrip/docs/Follow/FollowNightscout.html
